PowerUp-2018/src/com/spikes2212/robot/Commands/commandGroups/MoveAndLockLift.java

```diff
@@ -0,0 +1,21 @@
+package com.spikes2212.robot.Commands.commandGroups;
+
+import com.spikes2212.genericsubsystems.commands.MoveBasicSubsystem;
+import com.spikes2212.robot.Robot;
+import com.spikes2212.robot.SubsystemConstants;
+import com.spikes2212.robot.Commands.MoveLift;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class MoveAndLockLift extends CommandGroup {
+
+	//The variable moveUp tells the command group if the lift should move up or down
+	public MoveAndLockLift(boolean moveUp) {
+		addSequential(new MoveBasicSubsystem(Robot.liftLocker, SubsystemConstants.LiftLocker.UNLOCK_SPEED));
```

Comment:
instead of doing it like this, just get the speed in the constructor and use it on the lift